gematria/granite/python/token_graph_builder_model.py

```diff
@@ -378,8 +366,9 @@ def _make_batch_feed_dict(self) -> model_base.FeedDict:
     return feed_dict
 
 
-class TokenGraphBuilderModelNodeEmbed(snt.Embed):
-  """Extends `snt.Embed` to include instruction annotations in node embeddings.
+class TokenGraphBuilderModelNodeEmbed:
+  """`snt.Embed`-like class representing node embeddings with instruction
+  annotations included.
```

Comment:
Very nitpicky: the first line of the docstring should fit into 80 columns. I think we can move the note about `snt.Embed` to the extended part of the docstring.